tests/unit/test_security.py

```diff
@@ -0,0 +1,130 @@
+from asynction.security import build_http_api_key_security_check
+from asynction.security import build_http_security_check
+from asynction.security import build_oauth2_security_check
+from asynction.security import build_security_handler
+from asynction.security import load_api_key_info_func
+from asynction.security import load_basic_info_func
+from asynction.security import load_token_info_func
+from asynction.types import HTTPAuthenticationScheme
+from asynction.types import OAuth2Flow
+from asynction.types import OAuth2Flows
+from asynction.types import SecurityScheme
+from asynction.types import SecuritySchemesType
+from tests.fixtures import FixturePaths
+
+
+def test_load_basic_info_func():
+    scheme = SecurityScheme(
+        SecuritySchemesType.HTTP,
+        scheme=HTTPAuthenticationScheme.BASIC,
+        x_basic_info_func="tests.fixtures.handlers.basic_info",
+    )
+    basic_info = load_basic_info_func(scheme)
+    assert basic_info
+    assert callable(basic_info)
+
+
+def test_load_api_key_info_func(fixture_paths: FixturePaths):
+    scheme = SecurityScheme(
+        SecuritySchemesType.HTTP_API_KEY,
+        name="api_key",
+        in_="query",
+        x_api_key_info_func="tests.fixtures.handlers.api_key_info",
+    )
+    api_key_info = load_api_key_info_func(scheme)
+    assert api_key_info
+    assert callable(api_key_info)
+
+
+def test_load_token_info_func(fixture_paths: FixturePaths):
+    scheme = SecurityScheme(
+        SecuritySchemesType.OAUTH2,
+        flows=OAuth2Flows(implicit=OAuth2Flow(authorization_url="", scopes={"a": "a"})),
+        x_token_info_func="tests.fixtures.handlers.token_info",
+    )
+    token_info = load_token_info_func(scheme)
+    assert token_info
+    assert callable(token_info)
```

Comment:
here we should check that the `token_info` the actual method (after importing it)

for example:

```python
from test.fixtures import handlers

def test_load_token_info_func(fixture_paths: FixturePaths):
    scheme = SecurityScheme(
        SecuritySchemesType.OAUTH2,
        flows=OAuth2Flows(implicit=OAuth2Flow(authorization_url="", scopes={"a": "a"})),
        x_token_info_func="tests.fixtures.handlers.token_info",
    )
    assert load_token_info_func(scheme) == handlers.token_info
```

And similar for the rest of the loaders.